src/Subscriptions/Serializer.php

```diff
@@ -63,7 +66,16 @@ public function unserialize(string $context): GraphQLContext
 
         $request->setUserResolver(
             function () use ($rawUser) {
-                return unserialize($rawUser);
+                $user = $this->getRestoredPropertyValue($rawUser);
+
+                // This is here for backwards compatibility, before the Laravel
```

Comment:
To clarify, this is only needed so that current deployments can be upgraded seamlessly?

Reply:
Correct.

Reply:
Cool, nicely done. Can you change the wording to reflect that this enables seamless upgrades to existing deployments and add a TODO to remove this in the next major version?